content/404.md

```diff
@@ -7,3 +7,5 @@ permalink: 404.html
 We couldn't find what you were looking for.
 
 Please contact the owner of the site that linked you to the original URL and let them know their link is broken.
+
+[Back to home ](https://reactjs.org)
```

Comment:
Seems like these changes are not visible in the preview. This is the right file for the 404 page https://github.com/reactjs/reactjs.org/blob/main/src/pages/404.js.

Reply:
Ok i will fix the issue

Reply:
If this has no effect or use it can be removed